cmd/dump.go

```diff
@@ -95,14 +94,15 @@ func dumpMeta(m meta.Meta, dst string, threads int, keepSecret, fast, skipTrash
 		}()
 
 		if strings.HasSuffix(dst, ".gz") {
-			zw := gzip.NewWriter(fp)
+			w, _ = gzip.NewWriterLevel(fp, gzip.BestSpeed)
+			defer func() {
+				err = errors.Join(err, w.Close())
```

Comment:
w may be nil

Reply:
It's always non-nil as long as the 'level' is valid.